build-system/tasks/extension-generator/index.js

```diff
@@ -303,9 +295,9 @@ module.exports = {
 
 makeExtension.description = 'Create an extension skeleton';
 makeExtension.flags = {
-  name: '  The name of the extension. Preferable prefixed with `amp-*`',
+  name: '  The name of the extension. Should be prefixed with `amp-*`',
   bento: '  Generate a Bento component',
-  version: '  Sets the verison number (default: 1.0); --bento only',
+  version: '  Sets the verison number (default: 0.1; or 1.0 with --bento)',
```

Comment:
@rcebulko I forgot to specify this when I made `--version` work with legacy extensions.

Reply:
Nice update; thanks for backporting some of these things into the legacy generator!